src/View/mainpage/MainPage.py

```diff
@@ -89,37 +86,21 @@ def setup_central_widget(self):
         self.left_panel.setMaximumWidth(500)
 
         # Add structures tab to left panel
-        if patient_dict_container.has_modality("rtss"):
-            # Only add structures if ROIs exist, or there are no ROIs
-            # but the rtss is marked as being modified
-            if len(patient_dict_container.get("rois")) > 0 or\
-                (len(patient_dict_container.get("rois")) == 0\
-                 and patient_dict_container.get("rtss_modified") is True):
-                self.structures_tab = StructureTab()
-                self.structures_tab.request_update_structures.connect(
-                    self.update_views)
-                self.left_panel.addTab(self.structures_tab, "Structures")
-        elif hasattr(self, 'structures_tab'):
-            del self.structures_tab
+        if not hasattr(self, 'structures_tab'):
+            self.structures_tab = StructureTab()
+            self.structures_tab.request_update_structures.connect(self.update_views)
```

Comment:
```suggestion
            self.structures_tab.request_update_structures.connect(
                self.update_views)
```

PEP 8 compliance - line too long (also line 99).